### PR TITLE
Async ColorPicker and Slider

### PR DESCRIPTION
**Pull request by Marc-Andre-Rivet**
Followup to https://github.com/plotly/dash/pull/973

** Finalizing this will require `dash==1.5.0` with async support to be released and updating the `@plotly/webpack-dash-dynamic-import` plugin to `1.1.0`

### CODE REVIEW

**Comment by Marc-Andre-Rivet** [comment]
Side-quest for later: https://github.com/plotly/dash-daq/issues/71 -- as this project doesn't have CI setup

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a
 Changelog](http://keepachangelog.com/en/1.0.0/) and this project
 adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.3.0] - 2019-11-05
+### Added
+* [#70](https://github.com/plotly/dash-daq/pull/70) Async ColorPicker and Slider components
```
Comment (shammamah) [inline CHANGELOG.md]: We should probably use this sort of format (with relevant PR/issue # linked) for all CHANGELOG entries going forward! I'll make sure to do this in the future.

File: .eslintrc [hunk 2]
```diff
@@ -1,2 +1,10 @@
----
-  extends: ./node_modules/dash-components-archetype/config/eslint/eslintrc-react.json
+{
+  "extends": [
+    "./node_modules/dash-components-archetype/config/eslint/eslintrc-react.json"
```
Comment (alexcjohnson) [inline .eslintrc]: oy, not blocking but can we get rid of archetype at some point?

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
 --index-url https://pypi.python.org/simple/
 
 -e .
-dash==1.0.0
+dash>=1.5.1
```
[the remark this answers is not in the document]
Comment (shammamah) [reply requirements.txt]: (`dash>=1.5.1` should probably go into the `install_requires` in `setup.py` instead)

[the remark this answers is not in the document]
Comment (Marc-Andre-Rivet) [reply requirements.txt]: Sure, I can bump to `1.6.0` if it's just for demo purposes

File: .eslintrc (hunk 2 above)
Comment (Marc-Andre-Rivet) [reply .eslintrc]: https://github.com/plotly/dash-daq/issues/72

**Pull request merged**